types/sequelize/index.d.ts

```diff
@@ -5105,6 +5105,27 @@ declare namespace sequelize {
     interface Options {
 
         /**
+         * The username which is used to authenticate against the database.
```

Comment:
Having 3 overloads is preferable if the constructor takes different options under different circumstances.